src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -580,7 +580,7 @@ def load_arguments(self, _):
             c.argument('tags', tags_type)
             c.argument('no_wait', help='Do not wait for the long-running operation to finish.')
             c.argument('validate', options_list=['--validate'], help='Generate and validate the ARM template without creating any resources.', action='store_true')
-            c.argument('size', help='The VM size to be created. See https://azure.microsoft.com/pricing/details/virtual-machines/ for size info.')
+            c.argument('size', help='The VM size to be created. See https://azure.microsoft.com/pricing/details/virtual-machines/ for size info. Values from: az vm list-sizes.')
```

Comment:
Do you have default value for vm size to be created? If yes, you should declare here. And it will be better to say "Allowed values can be get through `az vm list-sizes` command."

Reply:
Looks like we should also change the description for populator-commands

Reply:
> 
> 
> Do you have default value for vm size to be created? If yes, you should declare here. And it will be better to say "Allowed values can be get through `az vm list-sizes` command."

Default value has been configured, but not here. "help" parameter's content is not the final output message. The format is copied from other parameters like --image.